src/components/PracticeAdvice/PracticeAdvice.jsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable max-len */
+import React from 'react';
+import './PracticeAdvice.style.scss';
+import TuneIcon from '@material-ui/icons/Tune';
+import SentimentSatisfiedIcon from '@material-ui/icons/SentimentSatisfied';
+import DoneAllIcon from '@material-ui/icons/DoneAll';
+import CreditCardIcon from '@material-ui/icons/CreditCard';
+
+function PracticeAdvice() {
+  return (
+    <section>
+      <div className="practiceContainer">
+        <div className="practiceHeader">
+          <h5>Practice Advice</h5>
+          <h1>Make online education accesible</h1>
+          <p>
+            Problems trying to resolve the conflict between
+            <br />
+            the two major realms of Classical physics: Newtonian mechanics
+          </p>
+        </div>
+        <div className="practiceBlocks">
+          <div className="block1">
+            <div className="iconBox1">
```

Comment:
call blocks descriptively, not by its number. Or atleast spell the numbers. It would be nice to use different notation, like BEM for classes that are not in JSS. Ofc you shouldn't find mix of those in typical project but we re learning. So for example you may want to use ```<div className='first-icon__box' or 'icon__box icon__box--first'>```